community/deb.sh

```diff
@@ -79,17 +79,47 @@ detect_os ()
   echo "Detected operating system as $os/$dist."
 }
 
+pgdg_check ()
+{
+  echo "Checking for postgresql-9.5..."
+  apt-cache show postgresql-9.5 2> /dev/null
+  if [ $? != 0 ]; then
+    pgdg_list="/etc/apt/sources.list.d/pgdg.list"
+    if [ -f $pgdg_list ]; then
+      cat <<EOM
+
+Unable to install PostgreSQL Apt Repository
+
+The file $pgdg_list already exists.
+
+Contact engage@citusdata.com with information about your system for help.
+
+EOM
+      exit 1
+    fi
+
+    echo "Installing PostgreSQL Apt Repository (PGDG)..."
+    echo "deb http://apt.postgresql.org/pub/repos/apt/ $(lsb_release -cs)-pgdg main" > $pgdg_list
```

Comment:
Ran into an awkward thing. If the `lsb-release` package is not installed prior to running this script then the `$dist` variable is set to 8, otherwise to `jessie`. We need the codename when running the deb command. One solution is to install the package prior to running `detect_os`...